### PR TITLE
chore: remove unused dependencies

### PR DESCRIPTION
**Pull request by marcelltoth**
That pile of dependabot PRs made me realize we list a lot of dependencies we don't actually use.

This PR removes them. I tested both `yarn start` and `yarn build` after each, but be sure to check it out and let me know if anything broke.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -27,7 +27,6 @@
     "@angular/platform-browser-dynamic": "~10.0.11",
     "@angular/router": "~10.0.11",
     "@stoplight/elements-web-components": "^6.0.0-beta.166",
-    "rxjs": "~6.6.2",
```
Comment (mallachari) [inline package.json:30]: Isn't rxjs used by angular itself?

**Review by mallachari — COMMENTED** [review]
I started fresh and it didn't build it correctly for me. Running with `yarn start` has similar effect.

Screenshot from 2020-10-16 13-06-40

I've noticed it doesn't run `ngcc --tsconfig './tsconfig.app.json'` during `yarn` for this branch. That might be the reason?

File: package.json (hunk 1 above)
Comment (marcelltoth) [reply package.json:30]: It is, but it's not a peer dependency, so doesn't matter.
image

**Comment by marcelltoth** [comment]
> I've noticed it doesn't run ngcc --tsconfig './tsconfig.app.json' during yarn for this branch. That might be the reason?

I've noticed that as well. I don't know where that feature vanished to, but I guess it's unrelated to this PR. Does it work if you run `ngcc` by hand? And how come it works on CI btw? :D

**Comment by mallachari** [comment]
> > I've noticed it doesn't run ngcc --tsconfig './tsconfig.app.json' during yarn for this branch. That might be the reason?
> 
> I've noticed that as well. I don't know where that feature vanished to, but I guess it's unrelated to this PR. Does it work if you run `ngcc` by hand? And how come it works on CI btw? :D

Yes, it actually does work after running `ngcc --tsconfig './tsconfig.app.json' ` by hand. Interesting :thinking:

**Comment by marcelltoth** [comment]
@mmiask @mpodlasin what is your experience with the above?

**Comment by mmiask** [comment]
@marcelltoth nothing unusual on my side:

Screenshot 2020-10-19 at 11 11 58

Screenshot 2020-10-19 at 11 11 43

**Comment by marcelltoth** [comment]
@stoplightio/void-crew now is anyone willing to approve, or request changes or do stuff? 😅

**Comment by mmiask** [comment]
> @stoplightio/void-crew now is anyone willing to approve, or request changes or do stuff? 😅

@mallachari  does the build still fail on your side?

**Comment by mallachari** [comment]
> > @stoplightio/void-crew now is anyone willing to approve, or request changes or do stuff? 😅
> 
> @mallachari  does the build still fail on your side?

I'll check in a moment but if nothing changes I guess it does. Though it looks like it's issue on my side as it works for all of you. Anyway I'll check it.

**Comment by mallachari** [comment]
I still have that ngcc error. Maybe let's just add that step to build process?

**Comment by marcelltoth** [comment]
> Maybe let's just add that step to build process?

I can recall you already did that some time ago. I don't know where that went, but let's fix it. It's weird though, the [docs](https://angular.io/guide/ivy#ivy-and-universalapp-shell) say it should only be needed for non-default settings

**Comment by mallachari** [comment]
> > Maybe let's just add that step to build process?
> 
> I can recall you already did that some time ago. I don't know where that went, but let's fix it. It's weird though, the [docs](https://angular.io/guide/ivy#ivy-and-universalapp-shell) say it should only be needed for non-default settings

Can't explain it, but it worked for me then and it works now. I guess chances that someone experiences that one day is quite high so better to have it. 
It was in `postinstall`. I added it back.

**Review by mallachari — APPROVED** [review]
Now it works good!

**Pull request merged**